### ISSUE TITLE
Two Zones Kiln -- 2 Heat Elements and 2 Thermocouples, possible?

### ISSUE THREAD

**Issue by agusting** [pvarney/PiLN#6]
Hi! first of all thank you for sharing your project. 
I have a Kiln with 2 heating elements and a controller that has to be set up mannually and would like to use PiLN to replace it so I can Program it. 
My Kiln has two zones with two thermocouples and would like to know if you think it could be easy to usel PiLN to control both Zones in order to have better control of the temperature.

Thanks in advance!

**Comment by pvarney** [comment]
Interesting.  The easiest thing to do off the top of my head would be to have 2 separate daemons, databases and web front ends. Are you looking to have different temperatures in the different zones? If not, it wouldn't be too hard to take the average of the thermocouples and have the heating element control be the same for both zones.

**Comment by pvarney** [comment]
And you're welcome and sorry for taking so long to get back to you.

**Comment by agusting** [comment]
Great!
I'll try to give it a shot.
I'll let you know how it goes.

Do you think it would be possible to run 2 Daemons from 1 RaspberryPi?

Regards


*Agustín Goya*
Editor (EDA)

WANCAMP | POST
tel. +54 11 4857 9083

cel. +54 9 11 15 6545 2427
Skype. agustingoya
agustin@wancamp.com.ar <andres@wancamp.com.ar>
wancamp.com.ar <http://www.wancamp.com.ar/>

**Comment by pvarney** [comment]
Yes, but I think you'd have to do a lot of renaming for each to work in its own. You could try running just one database, but I'm thinking it would have issues. If you do give it a try,  let me know how it turns out or if I can be of any help.
